### PR TITLE
[apsi] Update to 0.11.0

### PR DESCRIPTION
**Pull request by Thomas1664**
[apsi] Update to 0.11.0
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/apsi/portfile.cmake [hunk 1]
```diff
@@ -28,20 +28,18 @@ endif()
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
-    DISABLE_PARALLEL_CONFIGURE
     OPTIONS
-        "-DAPSI_BUILD_TESTS=OFF"
-        "-DAPSI_BUILD_CLI=OFF"
+        -DCMAKE_FIND_PACKAGE_PREFER_CONFIG=ON
```
Comment (Thomas1664) [inline ports/apsi/portfile.cmake:32]: Upstream misses `CONFIG` during every call to `find_package`: https://github.com/microsoft/APSI/blob/v0.11.0/CMakeLists.txt

File: ports/apsi/vcpkg.json [hunk 2]
```diff
@@ -51,7 +52,8 @@
     "zeromq": {
       "description": "Use ZeroMQ for networking.",
       "dependencies": [
-        "cppzmq"
+        "cppzmq",
+        "zeromq"
```
Comment (Thomas1664) [inline ports/apsi/vcpkg.json:56]: Upstream directly links with `zeromq`:
https://github.com/microsoft/APSI/blob/146070d95f5057977d7f10bb74a95ac97b3acb1a/CMakeLists.txt#L180
https://github.com/microsoft/APSI/blob/146070d95f5057977d7f10bb74a95ac97b3acb1a/CMakeLists.txt#L273

File: ports/apsi/fix-find-seal.patch [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 0951577..c5efa1e 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -117,7 +117,7 @@ set(APSI_INCLUDES_INSTALL_DIR ${CMAKE_INSTALL_INCLUDEDIR}/APSI-${APSI_VERSION_MA
+ # find_package might throw a FATAL_ERROR before"xxx: not found", e.g. with vcpkg.cmake.
+ 
+ # Microsoft SEAL
+-find_package(SEAL 4 QUIET REQUIRED)
++find_package(SEAL QUIET REQUIRED)
```
Comment (Thomas1664) [inline ports/apsi/fix-find-seal.patch:10]: Avoid baseline regressions like #29193

File: ports/apsi/portfile.cmake [hunk 4]
```diff
@@ -3,11 +3,12 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
...
     REPO microsoft/APSI
-    REF 2dff8dcd39c361527ea3b320f87cb8e71dd4f777 #0.9.0
-    SHA512 16c52642719f1d67dfaa70d963ba8795ac618f250752a1f95d91d4b1db8b51b2598999dcc9a9a7a3dbe8537943a3c3bf2ec684cd2697fca88135b01009961213
+    REF "v${VERSION}"
+    SHA512 ad49e169970921175728705c6f5fe8062e74900a601ea25e9f836840a221175bfbb3169717787b1ca7e2e1a0b26630690dcef9bc91c0123877b856ea927e9e68
     HEAD_REF main
     PATCHES
-        fix-find_package.patch
+        fix-find-seal.patch
+        fix-c2398.patch
```
Comment (LilyWangLL) [inline ports/apsi/portfile.cmake:11]: Does this patch need to submit on upstream?

Comment (Thomas1664) [reply ports/apsi/portfile.cmake:11]: Upstream PR: https://github.com/microsoft/APSI/pull/72

**Review by LilyWangLL — APPROVED** [review]

**Pull request merged**